### PR TITLE
crimson/os/seastore: teuthology experimental tests

### PR DESCRIPTION
**Pull request by xxhdx1985126**
Seastore's teuthology experimental tests can run successfully on our teuthology testbed with #55735 merged.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by xxhdx1985126** [comment]
jenkins test make check

**Comment by xxhdx1985126** [comment]
jenkins test api

**Review by athanatos — APPROVED** [review]

**Comment by xxhdx1985126** [comment]
jenkins test make check

**Comment by xxhdx1985126** [comment]
```
/home/jenkins-build/build/workspace/ceph-pull-requests/build/src/seastar/gen/src/proto/metrics2.pb.h:2838:16: error: call to member function 'Set' is ambiguous
  _impl_.name_.Set(static_cast<Arg_&&>(arg), args..., GetArena());
  ~~~~~~~~~~~~~^~~
/home/jenkins-build/build/workspace/ceph-pull-requests/src/seastar/src/core/prometheus.cc:81:16: note: in instantiation of function template specialization 'io::prometheus::client::LabelPair::set_name<const seastar::basic_sstring<char, unsigned int, 15, true>>' requested here
        label->set_name(ctx.label->key());
```

**Comment by xxhdx1985126** [comment]
jenkins test make check

**Comment by xxhdx1985126** [comment]
jenkins test make check

**Comment by xxhdx1985126** [comment]
jenkins test api

**Comment by xxhdx1985126** [comment]
jenkins test make check

File: qa/suites/crimson-rados-experimental/seastore/basic/clusters/fixed-1.yaml [hunk 1]
```diff
@@ -10,5 +10,11 @@ overrides:
     conf:
       osd:
         osd shutdown pgref assert: true
+      osd.0:
+        crimson seastar cpu cores: 0-0
+      osd.1:
+        crimson seastar cpu cores: 1-1
+      osd.2:
+        crimson seastar cpu cores: 2-2
```
Comment (cyx1231st) [inline qa/suites/crimson-rados-experimental/seastore/basic/clusters/fixed-1.yaml:18]: Might be better to validate with multiple cores.

Comment (athanatos) [reply qa/suites/crimson-rados-experimental/seastore/basic/clusters/fixed-1.yaml:18]: Yeah, I agree.

**Comment by cyx1231st** [comment]
Should we close this after https://github.com/ceph/ceph/pull/56150?

**Comment by xxhdx1985126** [comment]
> Should we close this after #56150?

Yeah, closing

**Pull request closed without merging**